docs/time_units.md

```diff
@@ -75,6 +75,10 @@ then you'd want the mutation rate in units of mutations per year.
 
 ## Coalescence, recombination, and migration rates in recapitation
 
+*Note: this section is relevant for nonWF SLiM simulations:
+in a single-species WF simulation, one tick = one generation,
```

Comment:
good catch on "single-species"; I guess multispecies models that use different timescales would need adjustment; do you intend to add discussion of that, or do you figure users doing that are advanced enough that they can infer what needs to be done?

Reply:
I don't want to mention that here, no. =)